incubator/hnc/internal/pkg/selectors/selectors.go

```diff
@@ -67,3 +73,24 @@ func getSelectorFromString(str string) (labels.Selector, error) {
 	}
 	return selector, nil
 }
+
+// GetNoneSelector returns true indicates that user do not want this object to be propagated
```

Comment:
Maybe add 'or the selector cannot parse so HNC default it to not propagate`?

Reply:
When I reviewed this in Ginny's repo, I suggested that if we return an error, the boolean value should be ignored, to be consistent with most other Go functions.

Reply:
Yeah I have updated according to Adrian's comment

Reply:
sg